### ISSUE TITLE
2.2.0 upgrade _unload function error

### ISSUE THREAD

**Issue by joaociocca** [Outpox/Bluetooth#11]
Right after upgrading to 2.2.0, there's a repetitive error for `_unload` function.

From `journalctl -f -n 100 -eu plugin_loader`:

```
Apr 22 00:07:40 steamdeck PluginLoader[1125]: [browser][INFO]: Installing Bluetooth from URL (Version: 2.2.0)
Apr 22 00:11:59 steamdeck PluginLoader[1125]: [browser][INFO]: Incrementing installs for Bluetooth from URL https://plugins.deckbrew.xyz/plugins (version 2.2.0)
Apr 22 00:16:19 steamdeck PluginLoader[1125]: [browser][INFO]: uninstalling Bluetooth
Apr 22 00:16:19 steamdeck PluginLoader[1125]: [browser][INFO]:  at dir /home/deck/homebrew/plugins/Bluetooth
Apr 22 00:16:19 steamdeck PluginLoader[1125]: [browser][INFO]: Installed Bluetooth (Version: 2.2.0)
Apr 22 00:16:19 steamdeck PluginLoader[1125]: ======== Running on http://127.0.0.1:1337 ========
Apr 22 00:16:19 steamdeck PluginLoader[1125]: (Press CTRL+C to quit)
Apr 22 00:16:19 steamdeck PluginLoader[1201]: [plugin][INFO]: Calling Loader unload function.
Apr 22 00:16:19 steamdeck PluginLoader[1201]: [plugin][INFO]: Attempting to unload with plugin Bluetooth's "_unload" function.
Apr 22 00:16:19 steamdeck PluginLoader[1201]: [plugin][INFO]: Could not find "_unload" in Bluetooth's main.py
Apr 22 00:16:19 steamdeck PluginLoader[1125]: [loader][INFO]: Loaded Bluetooth
Apr 22 00:16:19 steamdeck PluginLoader[1125]: [loader][INFO]: Loaded Bluetooth
Apr 22 00:16:19 steamdeck PluginLoader[1125]: [loader][INFO]: Loaded Bluetooth
Apr 22 00:16:19 steamdeck PluginLoader[1125]: [loader][INFO]: Loaded Bluetooth
Apr 22 00:16:19 steamdeck PluginLoader[1125]: [loader][INFO]: Loaded Bluetooth
Apr 22 00:16:19 steamdeck PluginLoader[1125]: [loader][INFO]: Loaded Bluetooth
Apr 22 00:16:19 steamdeck PluginLoader[1125]: [loader][INFO]: Loaded Bluetooth
Apr 22 00:16:19 steamdeck PluginLoader[1125]: [loader][INFO]: Loaded Bluetooth
Apr 22 00:16:19 steamdeck PluginLoader[1125]: [loader][INFO]: Loaded Bluetooth
Apr 22 00:16:20 steamdeck PluginLoader[1125]: [loader][INFO]: Loaded Bluetooth
Apr 22 00:16:20 steamdeck PluginLoader[46098]: [plugin][INFO]: Calling Loader unload function.
Apr 22 00:16:20 steamdeck PluginLoader[46098]: [plugin][INFO]: Attempting to unload with plugin Bluetooth's "_unload" function.
Apr 22 00:16:20 steamdeck PluginLoader[46098]: [plugin][INFO]: Could not find "_unload" in Bluetooth's main.py
Apr 22 00:16:21 steamdeck PluginLoader[46077]: [plugin][INFO]: Calling Loader unload function.
Apr 22 00:16:21 steamdeck PluginLoader[46077]: [plugin][INFO]: Attempting to unload with plugin Bluetooth's "_unload" function.
Apr 22 00:16:21 steamdeck PluginLoader[46077]: [plugin][INFO]: Could not find "_unload" in Bluetooth's main.py
Apr 22 00:16:21 steamdeck PluginLoader[46078]: [plugin][INFO]: Calling Loader unload function.
Apr 22 00:16:21 steamdeck PluginLoader[46078]: [plugin][INFO]: Attempting to unload with plugin Bluetooth's "_unload" function.
Apr 22 00:16:21 steamdeck PluginLoader[46078]: [plugin][INFO]: Could not find "_unload" in Bluetooth's main.py
Apr 22 00:16:21 steamdeck PluginLoader[46079]: [plugin][INFO]: Calling Loader unload function.
Apr 22 00:16:21 steamdeck PluginLoader[46079]: [plugin][INFO]: Attempting to unload with plugin Bluetooth's "_unload" function.
Apr 22 00:16:21 steamdeck PluginLoader[46079]: [plugin][INFO]: Could not find "_unload" in Bluetooth's main.py
Apr 22 00:16:21 steamdeck PluginLoader[46081]: [plugin][INFO]: Calling Loader unload function.
Apr 22 00:16:21 steamdeck PluginLoader[46081]: [plugin][INFO]: Attempting to unload with plugin Bluetooth's "_unload" function.
Apr 22 00:16:21 steamdeck PluginLoader[46081]: [plugin][INFO]: Could not find "_unload" in Bluetooth's main.py
Apr 22 00:16:21 steamdeck PluginLoader[46083]: [plugin][INFO]: Calling Loader unload function.
Apr 22 00:16:21 steamdeck PluginLoader[46083]: [plugin][INFO]: Attempting to unload with plugin Bluetooth's "_unload" function.
Apr 22 00:16:21 steamdeck PluginLoader[46083]: [plugin][INFO]: Could not find "_unload" in Bluetooth's main.py
Apr 22 00:16:21 steamdeck PluginLoader[46087]: [plugin][INFO]: Calling Loader unload function.
Apr 22 00:16:21 steamdeck PluginLoader[46089]: [plugin][INFO]: Calling Loader unload function.
Apr 22 00:16:21 steamdeck PluginLoader[46087]: [plugin][INFO]: Attempting to unload with plugin Bluetooth's "_unload" function.
Apr 22 00:16:21 steamdeck PluginLoader[46087]: [plugin][INFO]: Could not find "_unload" in Bluetooth's main.py
Apr 22 00:16:21 steamdeck PluginLoader[46089]: [plugin][INFO]: Attempting to unload with plugin Bluetooth's "_unload" function.
Apr 22 00:16:21 steamdeck PluginLoader[46089]: [plugin][INFO]: Could not find "_unload" in Bluetooth's main.py
Apr 22 00:16:21 steamdeck PluginLoader[46094]: [plugin][INFO]: Calling Loader unload function.
Apr 22 00:16:21 steamdeck PluginLoader[46094]: [plugin][INFO]: Attempting to unload with plugin Bluetooth's "_unload" function.
Apr 22 00:16:21 steamdeck PluginLoader[46094]: [plugin][INFO]: Could not find "_unload" in Bluetooth's main.py
```

**Comment by Outpox** [comment]
What Steam Client version are you running?  
I tested stable and beta and did not face this error. 🤔  
  
EDIT: It has nothing to do with what's changed but decky-loader is expecting an unload function to remove the plugin from what I can read. I'll have a look.

**Comment by Outpox** [comment]
Will close this issue once it's merged in Decky.

**Comment by joaociocca** [comment]
Glad to be of help <3

**Comment by Outpox** [comment]
Fixed in 2.2.1 that was just [released](https://github.com/SteamDeckHomebrew/decky-plugin-database/pull/607).  
Thanks for the bug report!
